packages/apollo-tools/jest.config.js

```diff
@@ -0,0 +1,9 @@
+const baseConfig = require("../../jest.base");
+
+module.exports = {
+  ...baseConfig,
+  testPathIgnorePatterns: [
+    ...baseConfig.testPathIgnorePatterns,
+    "snapshotSerializers"
```

Comment:
This wasn't in the original config. since they're under `__tests__` wouldn't they be ignored anyway?

Reply:
I think `__tests__` files _won't_ be ignored, however I think you're correct that this is unnecessary since they don't end in `.test.*` or `.spec.*`! Removing this worked for me locally, so trying in CI.

Reply:
ahh, I thought that was ignored in the base config 🙈